templates/taxi/driver_list.html

```diff
@@ -4,7 +4,13 @@
   <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
-      {% for driver in driver_list%}
+      {% for driver in driver_list %}
+        {% if driver.id == request.user.id %}
```

Comment:
The driver's username is correctly linked to the Driver detail page. Ensure that the URL pattern name 'taxi:driver-detail' is correctly defined in your urls.py file.